rholang/src/main/scala/rholang/rosette/Roselang.scala

```diff
@@ -281,14 +276,20 @@ extends StrFoldCtxtVisitor {
   import RosetteOps._
 
   def theTupleSpaceVar : String
-  def TS() = V("'t") // TODO: Replace with V( theTupleSpaceVar ) ?
+  def TS() = V("t") // TODO: Replace with V( theTupleSpaceVar ) ?
   def CH() = V( theCtxtVar )
   def H() = HV( theCtxtVar )
   def Here() = Some( HV( theCtxtVar ) )
+
+  // TODO: Swap with something crypotgraphically secure and ensure Fresh and FreshSymbol are of uniform length
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/util/UUID.html#randomUUID() says it uses a cryptographically random number generator, but doesn't matter how it gets the seed.

Reply:
I'm going to change the comment to "Review crypotgraphically secureness" as I'm not sure about the length of the output.

Reply:
If we used the whole UUID, that wouldn't be a problem, but you're truncating it.  So yeah, file a ticket to figure out how many bits we need to avoid collisions.

Reply:
Yeah Greg wrote it to be truncated originally https://github.com/rchain/Rholang/commit/474470401c78fc98af4ef99ff2f25037c51176d6 but I'm not sure why. I'll ask him the reasons when I see him tomorrow. I've filed a ticket for now https://rchain.atlassian.net/browse/RHOL-27 .